mission-control/docs/playbooks/concepts/playbook.md

```diff
@@ -68,8 +68,71 @@ Playbook parameter defines a parameter that a playbook needs to run.
 | `required`    | Specify if the parameter is required                                                              | `bool`              |          |
 | `icon`        | Icon of parameter. See [icons](https://github.com/flanksource/flanksource-ui/tree/main/src/icons) | `string`            |          |
 | `description` | Description of the parameter.                                                                     | `string`            |          |
-| `type`        | Type of parameter.                                                                                | `string`            |          |
-| `properties`  | Properties of parameter.                                                                          | `map[string]string` |          |
+| `type`        | Type of parameter. _(Defaults to "text")_                                                         | `string`            |          |
+| `properties`  | Properties of parameter. _Varies based on the type_                                               | `map[string]string` |          |
+
+#### Parameter Type
+
+| name        | Description                        | UI Component | Schema    | Properties                                              |
+| ----------- | ---------------------------------- | ------------ | --------- | ------------------------------------------------------- |
+| `check`     | Limits the value to a check.       | Dropdown     | `string`  | [`Check Property`](#check-parameter-properties)         |
+| `checkbox`  | Boolean value toggle               | Checkbox     | `boolean` | -                                                       |
+| `code`      | Text area                          | Code Editor  | `string`  | [`Code Property`](#code-parameter-properties)           |
+| `component` | Limits the value to a component.   | Dropdown     | `string`  | [`Component Property`](#component-parameter-properties) |
+| `config`    | Limits the value to a config item. | Dropdown     | `string`  | [`Config Property`](#config-parameter-properties)       |
+| `list`      | Specify a custom list of values    | Dropdown     | `string`  | [`List Property`](#list-parameter-properties)           |
+| `people`    | Limits the value to people.        | Dropdown     | `string`  | [`People Property`](#people-parameter-properties)       |
+| `team`      | Limits the value to teams.         | Dropdown     | `string`  | -                                                       |
+| `text`      | Text input                         | Text Input   | `string`  | [`Text Property`](#text-parameter-properties)           |
+
+##### `check` parameter properties
+
+| Field    | Description                        | Schema   |
+| -------- | ---------------------------------- | -------- |
+| `filter` | Limit the checks to the given type | `string` |
```

Comment:
Isn't this a CEL expression ?

Reply:
How so? Isn't this a just a WHERE clause on the PostgREST call?